### ISSUE TITLE
Toggle italic fails 

### ISSUE THREAD

**Issue by funnym0nk3y** [dendronhq/dendron#751]
## Please select the area your Bug applys to. (Multiple selections are Possible)

- [ ] Onboard. Initial vault setup and import/export pods 
- [x] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [ ] Misc (Choose this if your not sure)

## Describe the bug

Dendron contributes the extension "Dendron Markdown Shortcuts", which contributes the command `toggle italic`. According to markdown best practises italics should be surrounded with one asterisk. Changing the setting to do that breaks the insertion of an empty italc character sequence. 

```
Running the contributed command: 'md-shortcut.toggleItalic' failed.
```

On the original extension repo this is known, but unfixed since 2019. 

## To Reproduce

1. Place the cursor somewhere, don't mark anything
2. Toggle italic

## Additional context

I know this isn't the original repo, but since there is no fix available since 2019 and dendron uses that extension, I thought of giving it a shot here. This raises the general question whether to base dendron of 3rd party extensions.

**Comment by kpathakota** [comment]
Yeah, this is a good find. Easy to repro but it is in a dependency. I think we will eventually want to update ours to handle this corner case. However the workaround of selecting something does seem to work.

**Comment by kpathakota** [comment]
@all-contributors add @funnym0nk3y for bug
